website/content/Resources/notebook.md

```diff
@@ -51,7 +64,7 @@ returns an error RESOURCE_ALREADY_EXISTS. If this operation fails it may have su
 
 #### - `format`:
 > **(Required)** This specifies the format of the file to be imported. 
-By default, this is SOURCE. However it may be one of: SOURCE, HTML, JUPYTER, DBC. The value is case sensitive.
+By default, this is SOURCE. However it may be one of: SOURCE, HTML, JUPYTER, DBC. The value is case sensitive. Note: For Azure, JUPYTER differs from other options in producing a notebook in a file format supported by the [workspace import and export API](https://docs.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/workspace#--request-structure-1).
```

Comment:
Hey can you elaborate more on that statement. I do not quite understand it. I believe regardless of the cloud, the JUPYTER option works, it just requires the input notebooks to be in a .ipynb format. I think the statement should be more in regards to stating that, when using the JUPYTER format, the notebooks should be in the .ipynb extension, SOURCE should be either .scala, .py, .r, .sql extension based files, DBC should be extension .dbc, and HTML is .html. 

Please let me know if you have any questions.

Reply:
Hey, the documentation you linked was for an Azure API, so I specified this. I'll make the suggested update and remove the cloud-specific reference.